extensions/wikia/FounderEmails/FounderEmailsEvent.class.php

```diff
@@ -71,27 +75,32 @@ public function setData( Array $data ) {
 		$this->mData = $data;
 	}
 
-	abstract public function enabled ( $wikiId, User $user );
+	abstract public function enabled( User $admin, $wikiId = null );
 
 	public function enabled_wiki( $wgCityId ) {
 		$wikiService = ( new WikiService );
 		$user_ids = $wikiService->getWikiAdminIds( $wgCityId );
 		foreach ( $user_ids as $user_id ) {
 			$user = User::newFromId( $user_id );
-			if ( $this->enabled( $wgCityId, $user ) )
+			if ( $this->enabled( $user, $wgCityId ) )
 				return true;
 		}
 		return false;
 	}
 
-	public static function isAnswersWiki() {
```

Comment:
There was code that was acting as if it were wiki-agnostic and taking a wiki ID, and then would call this function that assumed the current wiki.  Changed that so that this function actually checks the ```wgEnableAnswers``` value on the wiki ID given .